iroh-net/src/hp/netcheck.rs

```diff
@@ -566,7 +566,13 @@ impl ReportState {
         };
 
         let pinger = if self.plan.has_https_probes() {
-            Some(Pinger::new().await.context("failed to create pinger")?)
+            match Pinger::new().await {
+                Ok(pinger) => Some(pinger),
+                Err(err) => {
+                    debug!("failed to create pinger: {err:#}");
```

Comment:
Maybe remove the https probes from the plan in this case?

Reply:
hum, no i think they should stay in since they will keep working.  I'm not really sure why these pingers are triggered by the https probes in the plan, the tailscale code only uses the https and pinger probes if the udp stun fails.  but i decided that changing (or even investigating this) should not be mixed into this PR.

but good call, should create an issue not to forget: https://github.com/n0-computer/iroh/issues/1138

Reply:
Maybe the pinger should be created lazily? The reason is that I went and finished a todo that they had, where the https probes are actually integrated into the plans, instead of being added manually after the fact.

Reply:
Yes, can we do this as part of #1138? In either case the fix of making failure to create the pinger non-fatal is needed.

Reply:
sure